pkg/types/manifest.go

```diff
@@ -29,6 +29,7 @@ type VersionManifest struct {
 	DataExchange      *ManifestEntry `json:"dataexchange-https"`
 	TokensERC1155     *ManifestEntry `json:"tokens-erc1155"`
 	TokensERC20ERC721 *ManifestEntry `json:"tokens-erc20-erc721"`
+	Sandbox           *ManifestEntry `json:"sandbox"`
```

Comment:
I don't think `Sandbox` actually needs to be in the manifest struct at this point. I can't see that it's used anywhere. Can we remove it?

Reply:
Good callout, thanks!